src/hyperloglog.c

```diff
@@ -393,7 +393,7 @@ static char *invalid_hll_err = "-INVALIDOBJ Corrupted HLL object detected";
 /* Our hash function is MurmurHash2, 64 bit version.
  * It was modified for Redis in order to provide the same result in
```

Comment:
```suggestion
 * It was modified for Valkey in order to provide the same result in
```

Reply:
Sure, I will update the comments in the changed files at once.